From 4d1633d966ba4ffcb376aaf7bc0c10ffaf54a237 Mon Sep 17 00:00:00 2001
From: xianming wang <mingwax@codeaurora.org>
Date: Wed, 5 Sep 2018 11:12:49 +0800
Subject: [PATCH 186/194] SnapdragonCamera: Fix parameters NullPointerException

Fix parameters is NullPointerException in China App Stress.

CRs-Fixed: 2307806
Change-Id: I71fff4cb32361ad0e66d6460ca8dc01167c8a009
---
 src/com/android/camera/util/CameraUtil.java | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/com/android/camera/util/CameraUtil.java b/src/com/android/camera/util/CameraUtil.java
index 743146504..3682c80f4 100755
--- a/src/com/android/camera/util/CameraUtil.java
+++ b/src/com/android/camera/util/CameraUtil.java
@@ -188,6 +188,7 @@ public class CameraUtil {
     }
 
     public static boolean isVideoSnapshotSupported(Parameters params) {
+        if (params == null) return false;
         return TRUE.equals(params.get(VIDEO_SNAPSHOT_SUPPORTED));
     }
 
-- 
2.17.1

